packages/qdrant-loader/src/qdrant_loader/core/async_ingestion_pipeline.py

```
@@ -302,15 +300,19 @@ async def cleanup(self):
             except Exception as e:
-                logger.warning(f"Error stopping metrics server: {e}")
+                logger.warning(
+                    f"Error stopping metrics server: {sanitize_exception_message(e)}"
+                )
 
             # Use resource manager for cleanup
             if hasattr(self, "resource_manager"):
                 await self.resource_manager.cleanup()
 
             logger.info("Pipeline cleanup completed")
         except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)